frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx

```diff
@@ -784,9 +784,7 @@ export const AlertConfigurationCard = ({
 											className={styles.channelSelect}
 											options={channels}
 											mode="multiple"
-											onFocus={() => {
-												syncSlack()
-											}}
```

Comment:
@shayneo I prefer this as well, but followed the pattern I saw in `onSearch` - thanks for cleaning it up 😸